packages/angular_devkit/build_angular/package.json

```diff
@@ -52,7 +52,6 @@
     "sass-loader": "8.0.2",
     "semver": "7.1.3",
     "source-map": "0.7.3",
-    "source-map-support": "0.5.16",
```

Comment:
Oops, please add this back as this is a direct depedency of `build-angular`.

Reply:
Done. Any particular reason this is needed but not detected? I'm not sure how `yarn` decided to remove this.

Reply:
I am aware of any functionality from yarn to remove dependencies.

Reply:
Hmm, strange. I'm not sure why these got removed/reordered then. I think this happened when I did `yarn add`, but I'm not too sure as it was some months ago when I first made the PR.